src/nectarchain/dqm/charge_integration.py

```diff
@@ -1,13 +1,13 @@
 import ctapipe.instrument.camera.readout
 import numpy as np
 from ctapipe.coordinates import EngineeringCameraFrame
-from ctapipe.image.extractor import FixedWindowSum  # noqa: F401
-from ctapipe.image.extractor import FullWaveformSum  # noqa: F401
-from ctapipe.image.extractor import LocalPeakWindowSum  # noqa: F401
-from ctapipe.image.extractor import NeighborPeakWindowSum  # noqa: F401
-from ctapipe.image.extractor import SlidingWindowMaxSum  # noqa: F401
-from ctapipe.image.extractor import TwoPassWindowSum  # noqa: F401
-from ctapipe.image.extractor import GlobalPeakWindowSum
+from ctapipe.image.extractor import FixedWindowSum  # noqa
+from ctapipe.image.extractor import FullWaveformSum  # noqa
+from ctapipe.image.extractor import GlobalPeakWindowSum  # noqa
+from ctapipe.image.extractor import LocalPeakWindowSum  # noqa
+from ctapipe.image.extractor import NeighborPeakWindowSum  # noqa
+from ctapipe.image.extractor import SlidingWindowMaxSum  # noqa
+from ctapipe.image.extractor import TwoPassWindowSum  # noqa
```

Comment:
Why are all quality assurance checks skipped ?

Reply:
In this module the extractor can be called during execution through the `eval` statement. We then need to import those modules but, because of there are not explicitly called, flake don't like it.

Reply:
Using `eval` is not the cleanest way to code, but, I don't see a fast way to rewrite the module

Reply:
I didn't know it was possible to write this, I change